javascripts/components/cartModal.js

```diff
@@ -0,0 +1,83 @@
+import { cartTotal } from './cartTotal.js';
+import { makeCartModal } from './makeCart.js';
+import { getCart, emptyCart } from './../helpers/data/cartData.js';
+
+
+const makeCartModal = () => {
```

Comment:
It looks like you are importing this into itself? Where are you importing this to another file? Does the modal show in your application?

Reply:
It was working last night even after several hard resets but it is not showing up now. I will delete the import and see what happens.

Reply:
There is a new div with an ID that we added to the HTML last night. Maybe try targeting that for your modal?

Reply:
I went back and stepped through it from the beginning and made changes as I walked back  through it. Now it is working. I was not able to update the link in class tracker. Here. is the link to the new PR. https://github.com/wppattonjr/es6-modules-jquery/compare/master...cart-wireup